config/admins.txt

```diff
@@ -135,7 +135,7 @@ octareenroon91 = Game Master
 Serpentarium = Game Master
 Averagejoe82 = Game Master
 The Dreamweaver = Game Master
-88Naoki = Game Master

```

Comment:
No removing other people

Reply:
I recently changed my ckey from 88Naoki to Denton-30. @imsxz can confirm it